code/modules/spells/spell_types/voice_of_god.dm

```diff
@@ -0,0 +1,38 @@
+/obj/effect/proc_holder/spell/voice_of_god
+	name = "Voice of God"
+	desc = "Speak with an incredibly compelling voice, forcing listeners to obey your commands."
+	charge_max = 1200 //variable
+	cooldown_min = 0
+	level_max = 1
+	clothes_req = 0
+	action_icon_state = "voice_of_god"
+	var/command
+	var/cooldown_mod = 1
+	var/power_mod = 1
+	var/list/spans = list()
+
+/obj/effect/proc_holder/spell/voice_of_god/can_cast(mob/user = usr)
+	if(!user.can_speak())
+		user << "<span class='warning'>You are unable to speak!</span>"
+		return FALSE
+	return TRUE
+
+/obj/effect/proc_holder/spell/voice_of_god/choose_targets(mob/user = usr)
+	perform(user=user)
+/obj/effect/proc_holder/spell/voice_of_god/perform(list/targets, recharge = 1, mob/user = usr)
```

Comment:
ugh, what is that user=usr, do spells really not get user passed through to them as a variable?